app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/InformationFragment.java

```diff
@@ -91,42 +88,35 @@ public void showDevInfoPage() {
 
     @Override
     public void showTwitter() {
-        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://twitter.com/DroidKaigi"));
-        if (intent.resolveActivity(getContext().getPackageManager()) == null)
-            return;
-        startActivity(intent);
+        showHtmlUrl("https://twitter.com/DroidKaigi");
     }
 
     @Override
     public void showFacebook() {
-        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://www.facebook.com/DroidKaigi/"));
-        if (intent.resolveActivity(getContext().getPackageManager()) == null)
-            return;
-        startActivity(intent);
+        showHtmlUrl("https://www.facebook.com/DroidKaigi/");
     }
 
     @Override
     public void showGitHubRepository() {
-        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/DroidKaigi/conference-app-2017/"));
-        if (intent.resolveActivity(getContext().getPackageManager()) == null)
-            return;
-        startActivity(intent);
+        showHtmlUrl("https://github.com/DroidKaigi/conference-app-2017/");
     }
 
     @Override
     public void showDroidKaigiWeb() {
-        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://droidkaigi.github.io/2017/"));
-        if (intent.resolveActivity(getContext().getPackageManager()) == null)
-            return;
-        startActivity(intent);
+        showHtmlUrl("https://droidkaigi.github.io/2017/");
     }
 
     @Override
     public void showYouTube() {
-        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://www.youtube.com/droidkaigi"));
-        if (intent.resolveActivity(getContext().getPackageManager()) == null)
+        showHtmlUrl("https://www.youtube.com/droidkaigi");
+    }
+
+    public void showHtmlUrl(String htmlUrl) {
+        Intent intent = IntentHelper.buildActionViewIntent(getContext(), htmlUrl);
```

Comment:
Just in case, could you please add url format check like `URLUtil.isNetworkUrl`?